### PR TITLE
Video support

### PR DESCRIPTION
**Pull request by onmyway133**
- This should fix https://github.com/hyperoslo/Lightbox/issues/79
- Configure video by using `videoURL`

```swift
LightboxImage(
        image: UIImage(named: "photo2")!,
        text: "",
        videoURL: NSURL(string: "https://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4")
      )
```

- Configure video handling

```swift
LightboxConfig.handleVideo
```

simulator screen shot 07 aug 2016 17 02 25

### CODE REVIEW

File: Source/LightboxImage.swift [hunk 1]
```diff
@@ -4,18 +4,21 @@ public class LightboxImage {
 
   public private(set) var image: UIImage?
   public private(set) var imageURL: NSURL?
+  public private(set) var videoURL: NSURL?
   public var text: String
 
   // MARK: - Initialization
 
-  public init(image: UIImage, text: String = "") {
+  public init(image: UIImage, text: String = "", videoURL: NSURL? = nil) {
```
Comment (onmyway133) [inline Source/LightboxImage.swift]: Perhaps `Image` should be renamed to `Media` or something else

**Comment by zenangst** [comment]
This looks good to me, what do you think @vadymmarkov ?

**Comment by vadymmarkov** [comment]
@onmyway133 Looks great!

**Comment by vadymmarkov** [comment]
Is't it the time to release `Lightbox`? 😄

**Comment by zenangst** [comment]
@vadymmarkov it sure is! 😁

**Pull request merged**